src/routes/tasks/components/TaskInputsDrawerSection.tsx

```diff
@@ -385,15 +385,19 @@ const TaskInputsDrawerSection = ({
                     ([
                         identifier,
                         { algoInput: algoInput, inputs: inputs },
-                    ]) => (
-                        <TaskInputSectionEntry
-                            key={identifier}
-                            identifier={identifier}
-                            algoInput={algoInput}
-                            inputs={inputs}
-                            parentTasks={parentTasks}
-                        />
-                    )
+                    ]) => {
+                        if (inputs.length > 0) {
```

Comment:
I think we should not set the test here, but rather that the TaskInputSectionEntry component should handle correctly the case of no inputs for this identifier (ex: l. 332 write if inputs.length > 0 {...} else {<Text>No values for this input</Text>}).
Otherwise it means nothing will be displayed for this input

Reply:
Interesting !
I see the inputs from the algo as the list of the existing fields of the task (*). As such all those fields deserve to be displayed, even if there is no value plugged into it. We do not filter out the fields with null values do we ?
And actually it can ring a bell to the user, is it normal that a given input is not given any value ?

But the PR is merged so I guess we should just wait for users feedback now.

(*) to be more precise it is the list of "pluggable" input fields.

Reply:
I agree with @alexisdeh. Considering that every inputs will be setting up by the user, we need to display them into the listing to be sure that this input was well registered.

Having no value passed to this input is a second topic and showing that an input is well registered but nothing was passed into is a necessary information we need to show to the user. When there is no data, I would say to go with a simple "-" displayed, instead of a full error text or an empty space.